bins/argon/init.luau

```diff
@@ -4,4 +4,6 @@
 
 -- See https://github.com/lune-org/lune/pull/247 for more details
 
-require("./lune_packages/core")("argon-rbx/argon", _G.PESDE_ROOT)
+require("@self/lune_packages/toolchainlib")("argon-rbx/argon", _G.PESDE_ROOT)
+
+return nil
```

Comment:
Why return `nil`? Since the lune scheduler panic bug might have been fixed, we could replace this to take the exit code returned by the function back and call `process.exit` with that.

For example:

```luau
local exitCode = require("@self/lune_packages/toolchainlib")("argon-rbx/argon", _G.PESDE_ROOT)

return process.exit(exitCode)
```

Instead of repeating this code again, we could consider having a skeleton format and generate it with a lune script in the future.